source/docs/power-and-electronics/driver-station-guide.rst

```diff
@@ -1,13 +1,53 @@
 
-- A 5GHz phone is preferred over the standard 2.4GHz phone to reduce ping issues. Have you ever noticed that your team's robot seems to lag at competitions only?
+REV Driver Hub
+--------------
 
-Most teams run on 2.4GHz, which means that the 2.4GHz channels will be more crowded on competition day. This can lead to noticeably higher ping/lag which adversely affects driver and robot performance.
+The REV Robotics Driver Hub is effectively an Android phone with built in USB ports. It is purpose built for FTC, and contains several advantages over traditional Android phones.
 
-- USB and :term:`OTG <Micro USB On The Go (OTG) Cable>` connections can be a possible source of disconnects during a match. This is generally caused by poor quality connectors or adapters between the driver station and the gamepad controller. **It generally is worth it to purchase higher quality connectors as opposed to the cheapest options.**
+Advantages
+^^^^^^^^^^
+
+- Full sized USB ports do not require flimsy On The Go cables
+- Charges from USB-C including fast charging Power Delivery modes
+- Larger screen than many Android phones
+- Case is more robust then a standard Android phone
+
+Disadvantages:
+
+- Price: A Driver Hub costs $250.00, whereas legal Android phones can be purchased secondhand for less
+- Power Issues: Teams have reported random losses of power coming from an improperly sized battery.
+- Battery Issues: Teams have reported high battery drain in sleep mode, leading to hubs dying faster then expected
+- Wifi Issues: The wifi driver will occasionally crash when the device goes to sleep, requiring a device restart
+
+.. important::
+   If you purchase a Driver Hub and experience issues, refer to the `REV Driver Hub troubleshooting page <https://docs.revrobotics.com/duo-control/troubleshooting-the-control-system/driver-hub-troubleshooting>`_. The unexpected power off fix may be done pre-emptively to prevent a shutdown during a match
+
+Android Phones
+--------------
+
+As of 2023-2024, Android phones are still legal for competition use.
+
+Advantages
+^^^^^^^^^^
+
+- Android phones can be very cheap when purchased secondhand
+- Android phones generally are reliable out of the box
+
+Disadvantages
+^^^^^^^^^^^^^
+
+- The lack of a full size USB A port means that flimsy On The Go cables must be used
+- The phones themselves should be protected with a case to prevent damage
+
+A 5GHz phone is preferred over the standard 2.4GHz phone to reduce ping issues. Have you ever noticed that your team's robot seems to lag at competitions only?
+
+Most school wifi bands run on 2.4GHz, which means that the 2.4GHz channels will be more crowded on competition day. This can lead to noticeably higher ping/lag which adversely affects driver and robot performance.
```

Comment:
```suggestion
A 5 GHz phone is preferred over the standard 2.4 GHz phone to reduce ping issues. Have you ever noticed that your team's robot seems to lag at competitions only? This is because most school WiFi bands run on 2.4 GHz, which means that 2.4 GHz channels will be more crowded on competition day. This can lead to noticeably higher ping/lag which adversely affects driver and robot performance.
```